src/basic/rlimit-util.c

```diff
@@ -55,6 +61,39 @@ int setrlimit_closest(int resource, const struct rlimit *rlim) {
         return 0;
 }
 
+int setpriority_closest(int nice) {
+        int current, limit;
+        struct rlimit highest;
+
+        // Try to set requested nice level
```

Comment:
nitpick: we generally prefer /* C */ comments over // C++ comments, also see CODING_STYLE...

Reply:
Done.